src/main/java/de/sakpaas/backend/service/LocationDetailsRepository.java

```diff
@@ -1,11 +1,16 @@
 package de.sakpaas.backend.service;
 
 import de.sakpaas.backend.model.LocationDetails;
+import java.util.List;
 import java.util.Optional;
 import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
 
 public interface LocationDetailsRepository extends JpaRepository<LocationDetails, Long> {
 
   Optional<LocationDetails> findById(Long id);
 
+  @Query(value = "SELECT type FROM location_details GROUP BY type", nativeQuery = true)
```

Comment:
Wäre es nicht klüger hier mit DISTINCT zu arbeiten, da das ja genau dafür vorgesehen ist?

Reply:
Google sagt: https://blog.sqlauthority.com/2007/03/29/sql-server-difference-between-distinct-and-group-by-distinct-vs-group-by/

> A DISTINCT and GROUP BY usually generate the same query plan, so performance should be the same across both query constructs. GROUP BY should be used to apply aggregate operators to each group.

Ich würde es erstmal so lassen.

Reply:
Fabio zu liebe mache ich es trotzdem.

Reply:
Da steht ja auch:

> If all you need is to remove duplicates then use DISTINCT.

Aber danke C:

Reply:
Done